python/lsst/ctrl/mpexec/cli/script/pre_exec_init_qbb.py

```diff
@@ -55,14 +59,23 @@ def pre_exec_init_qbb(
         graph loaded from a file. Ignored if graph is not loaded from a file.
     config_search_path : `list` [`str`]
         Additional search paths for butler configuration.
+    **kwargs : `object`
+        Ignored; click commands may accept options for more than one script
+        function and pass all the option kwargs to each of the script functions
+        which ignore these unused kwargs.
     """
-    args = SimpleNamespace(
-        butler_config=butler_config,
-        qgraph=qgraph,
-        qgraph_id=qgraph_id,
-        config_search_path=config_search_path,
-    )
+    _LOG.verbose("Reading full quantum graph from %s.", qgraph)
+    # Load quantum graph. We do not really need individual Quanta here,
+    # but we need datastore records for initInputs, and those are only
+    # available from Quanta, so load the whole thing.
+    qg = QuantumGraph.loadUri(qgraph, graphID=BuildId(qgraph_id) if qgraph_id is not None else None)
```

Comment:
bps can specify on the command line one node ID for every PipelineTask if that provides enough information for initInputs.  The bps code is for this is still there from the non-qbb pipetaskInit work.  Presumably for huge graphs this would reduce memory requirements and speed up the loading.